### PR TITLE
Add contribute template

### PR DESCRIPTION
**Pull request by leonwanghui**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+<!--  Thanks for sending an issue!  Here are some tips for you:
+
+If this is your first time, please read our contributor guidelines: https://tinyms.readthedocs.io/en/latest/community/contributing.html
+-->
+
+## Environment
+
+- **TinyMS version (source or binary)**:
+- **MindSpore version (source or binary)**:
```
Comment (hannibalhuang) [inline .github/ISSUE_TEMPLATE/bug-report.md:16]: s/MindSpore version/Framework and version

**Review by hannibalhuang — CHANGES_REQUESTED** [review]
small nix

**Review by hannibalhuang — APPROVED** [review]
LGTM

**Review by lyd911 — APPROVED** [review]
lgtm

**Pull request merged**